docker-compose.yml

```
@@ -0,0 +1,9 @@
+version: '3'
+
+services:
+  mstk:
+    build: .
```

You can tag the image (with only a single tag, but still useful) so that you can do `docker run -it mstk` outside of the folder with `docker-compose.yml`.
```suggestion
    build: .
    image: mstk:latest
```

---

Good idea. I added this.